### PR TITLE
Add sersol_number_a to Solr

### PR DESCRIPTION
**Pull request by kazymovae**

### CODE REVIEW

File: lib/data/solr_fields_config.yml [hunk 1]
```diff
@@ -404,6 +404,10 @@ series_issn:
   type: num
 series_other_ids:
   type: e
+sersol_number_a:
```
Comment (corylown) [inline lib/data/solr_fields_config.yml:407]: Field name is just `sersol_number` without `_a`

Comment (kazymovae) [reply lib/data/solr_fields_config.yml]: Fixed.

File: lib/data/solr_fields_config.yml [hunk 2]
```diff
@@ -404,6 +404,10 @@ series_issn:
   type: num
 series_other_ids:
   type: e
+sersol_number:
+  type: e
+  attr:
+    stored
```
Comment (corylown) [inline lib/data/solr_fields_config.yml:410]: Sorry I missed this the first time around. Should be:

` - stored`

Comment (kazymovae) [reply lib/data/solr_fields_config.yml]: Done!

**Pull request merged**